src/ASTBuilder.ts

```diff
@@ -1616,7 +1616,9 @@ export class ASTBuilder
   public visitParameterList(
     ctx: SP.ParameterListContext
   ): (AST.VariableDeclaration & WithMeta)[] {
-    return ctx.parameter().map((paramCtx: any) => this.visitParameter(paramCtx))
+    return ctx
+      .parameter()
+      .map((paramCtx: SP.ParameterContext) => this.visitParameter(paramCtx))
```

Comment:
`ctx.parameter()` returns a `SP.ParameterContext[]` so we can assume that the map will iterate over `SP.ParameterContext`